rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/JobStateImpl.java

```diff
@@ -85,6 +88,19 @@ public String getOwner() {
     }
 
     @Override
+    public List<TaskState> getTaskByTag(String tag) {
+        ArrayList<TaskState> taskStateList = new ArrayList<>();
+        Map<String, TaskStateData> taskStateMap = d.getTasks();
+        for (TaskStateData ts : taskStateMap.values()) {
+            String taskTag = ts.getTag();
+            if(taskTag != null && taskTag.equals(tag)) {
```

Comment:
We will have to change the way tasks are handled by the JobState very soon (for the job/task decoupling effort), so I think it's not worth in the mean time to modify this method any further.I didn't know about the multimap and it seems great. I hope I'll have the opportunity to use it sometime.